src/WCSimTrackingAction.cc

```diff
@@ -41,6 +44,9 @@ WCSimTrackingAction::WCSimTrackingAction()
   ParticleList.insert(13); // mu-
   ParticleList.insert(-13); // mu+
   // don't put gammas there or there'll be too many
+  
+  ParticleList.insert(22);
+  ParticleList.insert(11);
```

Comment:
This also worries me in terms of increasing the output file size

Reply:
Same as with the ProcessList. Some time ago I needed to include those lines in order to see the gammas and electrons in my DataFrame, but now I tried with those lines commented and I see gammas and electrons just fine so I guess I don't really understand what this is doing.

What I do see are some jumps in the track_id in my DataFrame in the optical photons produced via Scintillation in the BGO. There are some optical photons that cannot be stored in my DataFrame and I don't know why.

Reply:
As mentioned by email, WCTE/WCSim#14 will give options to allow you to save all the tracks you need